src/backend/providers/playnite/PlayniteLaunchHelper.cpp

```diff
@@ -30,21 +30,25 @@ QString build_launch_cmd(const QString& exec_path, const QString& args)
 {
     return QStringLiteral("%1 %2").arg(QDir::cleanPath(exec_path), args);
 }
+} // namespace
+
+PlayniteLaunchHelper::PlayniteLaunchHelper()
+    : m_steam_call(providers::find_steam_call())
+{}
 
-PlayniteGameLaunchInfo::LaunchType get_launch_type_enum(const QString& type)
+PlayniteGameLaunchInfo::LaunchType PlayniteLaunchHelper::get_launch_type_enum(const int type)
 {
-    if (type.isEmpty()) {
+    if (type == 0) {
         return PlayniteGameLaunchInfo::LaunchType::INSTALLED;
     }
-    if (type == "1") {
+    if (type == 1) {
         return PlayniteGameLaunchInfo::LaunchType::STEAM;
     }
-    if (type == "2") {
+    if (type == 2) {
         return PlayniteGameLaunchInfo::LaunchType::EMULATOR;
     }
     return PlayniteGameLaunchInfo::LaunchType::UNKNOWN;
 }
```

Comment:
Since `type` is now a number, you could also use a `switch`, to shave off some code size.